docs/source/author/topics.rst

```diff
@@ -0,0 +1,685 @@
+Topics application
+##############################################################################
+
+The topics application is the main focus of the CS Unplugged website, as it
+contains the majority of educational material for the project.
+
+.. contents:: Contents
+  :local:
+
+Topics overview
+==============================================================================
+
+A general overview of the topics application can be described in the following
```

Comment:
Maybe just a note, so that they don't have to leave the page
> Recall: Django contains ‘applications’ which are Python packages that provide some set of features. Each large part/chunk of the CS Unplugged is a separate application.